addons/edi_stock/models/edi_move_request_record.py

```diff
@@ -165,3 +170,10 @@ def execute(self):
 
         # Cancel moves in bulk
         cancel._action_cancel()
+
+    @api.model
+    def log_progress(self, doc, log_message):
```

Comment:
May be worth putting _some_ of these on `edi.record` (`public/odoo_edi/addons/edi/models/edi_record.py`) so we don't need as many duplicate redefinitions.
I think we would still need one one for `EdiSaleRequestDocument` though